config/environments/test.rb

```diff
@@ -26,8 +26,8 @@
 
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
-  config.action_controller.perform_caching = false
-  config.cache_store = :null_store
+  config.action_controller.perform_caching = true            # originally false
+  config.cache_store = :memory_store, { size: 64.megabytes } # originally :null_store
```

Comment:
Enabling caching in all tests makes me nervous. But if we really want to do this, no need to have comments with the previous values. We can always look at the git history if we care.

Reply:
Removed the comments. Have to turn caching on since the test env will otherwise overwrite the cache-control header. I don't think there is anyway to just turn caching on for one test.

Reply:
Also, I fixed the tests as Mick suggested now.